internal/cloud/realm/client.go

```diff
@@ -77,6 +77,7 @@ type Client interface {
 	ClientTemplate(groupID, appID, templateID string) (*zip.Reader, bool, error)
 	CompatibleTemplates(groupID, appID string) ([]Template, error)
 
+	AllowedIPs(groupID, appID string) (AccessList, error)
```

Comment:
feel obligated to ask if we can either a) rename this to `AccessList(...)` or b) adjust the return signature to provide `([]AllowedIP, string, error)` (where the `string` is just the current IP)

feel free to tell me this makes the most sense semantically and we'll keep it as such, but otherwise I think I lean towards a) over b)

Reply:
Going with `AllowedIPs(groupID, appID string) ([]AllowedIP, error)` as per the other comment below.